tests/acceptance/16_cf-serverd/add_access_constraints_fail.cf

```diff
@@ -0,0 +1,45 @@
+body common control
+{
+  inputs => { "../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+  commands:
+    "/bin/echo -n '=line_number=' && /bin/grep -n GDB:.*add_access_constraints_fail.cf $(this.promise_dirname)/../../../cf-serverd/server_transform.c | cut -d: -f1"
```

Comment:
😲

Reply:
you think I should doc what it does a bit? It seems fairly straightforward for users of standard text processing commands.

Reply:
Haha "standard text processing commands". For the record, I think it's fine, but a documenting comment doesn't hurt.